core/device_hooks/common_kernels.inc.cpp

```diff
@@ -356,6 +356,18 @@ GKO_NOT_COMPILED(GKO_HOOK_MODULE);
 GKO_INSTANTIATE_FOR_EACH_VALUE_AND_INDEX_TYPE(
     GKO_DECLARE_CSR_MOVE_TO_DENSE_KERNEL);
 
+template <typename ValueType, typename IndexType>
+GKO_DECLARE_CSR_CONVERT_TO_SELLP_KERNEL(ValueType, IndexType)
+GKO_NOT_COMPILED(GKO_HOOK_MODULE);
+GKO_INSTANTIATE_FOR_EACH_VALUE_AND_INDEX_TYPE(
+    GKO_DECLARE_CSR_CONVERT_TO_SELLP_KERNEL);
+
+template <typename ValueType>
+GKO_DECLARE_CSR_CALCULATE_TOTAL_COLS_KERNEL(ValueType)
+GKO_NOT_COMPILED(GKO_HOOK_MODULE);
+GKO_INSTANTIATE_FOR_EACH_VALUE_TYPE(
```

Comment:
This kernel has both `ValueType` and `IndexType`. See [this CI error output](https://gitlab.com/ginkgo-project/ginkgo-public-ci/-/jobs/182435716)
```suggestion
GKO_INSTANTIATE_FOR_EACH_VALUE_AND_INDEX_TYPE(
```